osgar/record.py

```diff
@@ -6,15 +6,17 @@
 import sys
 import os
 import time
-from queue import Queue
+import signal
+import threading
 
 from osgar.logger import LogWriter
 from osgar.lib.config import load, get_class_by_name
 from osgar.bus import BusHandler
 
 
 class Recorder:
-    def __init__(self, config, logger, application=None):
+    def __init__(self, config, logger, application_class=None):
```

Comment:
Note, that original `application` keyword is used for example in `turn.py` (I may have too different branch) and that corresponding parameter name is used in `Replay` (yeah, that would need more refactoring). In general I would split "refactoring step" and "new feature step" (next time).